src/storage/gc_worker.rs

```diff
@@ -536,6 +536,7 @@ mod tests {
             .async_scan(
                 Context::default(),
                 Key::from_encoded_slice(b""),
+                None,
```

Comment:
em, can GC scan the stale data left by DeleteFilesInRanges?

Reply:
GC can touch these data, but it won't panic. So we don't need to worry about it too much, though it need to be fixed someday too. BTW, this PR is not related to GC.